api/app/files/api.py

```diff
@@ -45,9 +46,48 @@ def get(self):
             return send_file(temp.name, as_attachment=True, attachment_filename=args['filename'], mimetype='application/pdf')
 
 
+
+class FileAPIImproved(FileUploadMixin, restful.Resource):
+
+
+    @staticmethod
+    def get_by_id(file_id):
+        return db.session.query(File).get(file_id)

```

Comment:
Can we move this method into a separate repository file in `/api/app/files/repository.py` and into a class called `FileRepository`? Look at the rest of the project for examples. Then once it's in there, import the class in with `from app.files.repository import FileRepository as file_repository`. Then you will be able to use the method with `file_repository.get_by_id()`. The reason for this is that we're trying to separate the database operations from what goes on in the API file which results in cleaner code.